sktime/forecasting/enbpi.py

```diff
@@ -71,7 +77,7 @@ class EnbPIForecaster(BaseForecaster):
     >>> forecaster = Differencer(lags=[1]) * Deseasonalizer(sp=12) * EnbPIForecaster(
     ...    forecaster=NaiveForecaster(sp=12),
     ...    bootstrap_transformer=MovingBlockBootstrap(n_bootstraps=10))
-    >>> forecaster.fit(y, fh=range(12))
+    >>> forecaster = forecaster.fit(y, fh=range(1, 12)) # Range is broken
```

Comment:
what do you mean, range is broken - is there a bug?

Reply:
Was a note to me, that is now fixed. The original example was broken.